.claude/scripts/pm/in-progress.sh

```
@@ -0,0 +1,74 @@
+echo "==================="
+echo ""
+
+# Check for active work in updates directories
+found=0
+
+if [ -d ".claude/epics" ]; then
+  for updates_dir in .claude/epics/*/updates/*/; do
+    [ -d "$updates_dir" ] || continue
+
+    issue_num=$(basename "$updates_dir")
+    epic_name=$(basename $(dirname $(dirname "$updates_dir")))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)